app/src/components/Charts/TotalStreams/query.tsx

```diff
@@ -0,0 +1,15 @@
+import { TABLE } from '../../../db/queries/constants'
+
+export function query() {
+    return `
+SELECT
+  COUNT(*)::INTEGER AS count_streams,
+  SUM(ms_played)::INTEGER AS ms_played
```

Comment:
Error: Conversion Error: Type DOUBLE with value 34864968645.0 can't be cast because the value is out of range for the destination type INT32 when casting from source column sum(ms_played)